OHHTTPStubs/UnitTests/Test Suites/NilValuesTests.m

```diff
@@ -142,6 +170,32 @@ - (void)test_InvalidPath
     [self waitForExpectationsWithTimeout:kResponseTimeTolerence handler:nil];
 }
 
+- (void)test_InvalidPathWithURL
+{
+    [OHHTTPStubs stubRequestsPassingTest:^BOOL(NSURLRequest *request) {
+        return YES;
+    } withStubResponse:^OHHTTPStubsResponse *(NSURLRequest *request) {
+        return [[OHHTTPStubsResponse responseWithFileURL:[NSURL URLWithString:@"-invalid-url"] statusCode:501 headers:nil]
```

Comment:
Mmmh actually just realized that `[NSURL URLWithString:@"-invalid-url"]` will return `nil` and thus this test is kind of a duplicate of `test_NilPathWithURL`.

Probably better use a valid URL (as of the RFC, so it doesn't return `nil`) but that resolves to an invalid resource (non-existing file) instead?